forge-gui-mobile/src/forge/adventure/scene/QuestLogScene.java

```diff
@@ -77,8 +74,11 @@ private QuestLogScene() {
         ScrollPane scroller = new ScrollPane(scrollContainer);
         root.add(scroller).colspan(3).fill().expand();
 
-
-
+        backToListButton.addListener(new ClickListener() {
```

Comment:
try keep logic together:
instead of adding second listener make the right adjustments via the `ui.onButtonPress` call above